src/components/AlertCard.tsx

```diff
@@ -15,18 +15,18 @@ export function AlertCard({
     <div className="max-w-md" role="alert">
       <div
         className={cn(
-          "font-bold text-slate12 rounded-t px-4 py-2 mt-1",
-          style === "error" ? "bg-tomato9" : "bg-amber9",
+          "font-bold text-slate12 px-4 py-2 mt-1",
+          style === "error" ? "bg-tomato9" : "bg-amber9"
```

Comment:
prefer theme variable colors

Reply:
Notice that this component isn't being used. I will delete it.